src/scope.jl

```diff
@@ -117,16 +117,19 @@ function scopes(x::EXPR, state)
     s0 = state.scope
     if headof(x) === :file
         setscope!(x, state.scope)
+        add_eval_method(x, state)
```

Comment:
Why do we need to add this here? Is this for basically `Main.eval`?

Reply:
Yeah it is. I don't fully understand how `Main` is managed (what is `MainInclude` for?)